src/ray/gcs/gcs_server/gcs_resource_report_poller.h

```diff
@@ -33,23 +33,28 @@ class GcsResourceReportPoller {
 
  public:
   GcsResourceReportPoller(
-      std::shared_ptr<GcsResourceManager> gcs_resource_manager,
       std::shared_ptr<rpc::NodeManagerClientPool> raylet_client_pool,
       std::function<void(const rpc::ResourcesData &)> handle_resource_report,
+      // TODO (Alex): Ideally we should annotate this LOCKS_EXCLUDED(mutex_)
```

Comment:
Classic case of "Alex doesn't know C++". Would like to do this correctly before merging though.